docs/markdown/howtox.md

```diff
@@ -239,6 +239,26 @@ And then pass it through the variable (remember to use absolute path):
 $ SCANBUILD=$(pwd)/my-scan-build.sh ninja -C builddir scan-build
 ```
 
+## Use clippy
+
+If your project includes Rust targets, you can invoke clippy like this:
+
+```console
+$ meson setup builddir
+$ ninja -C builddir clippy
+```
+
+Clippy will also obey the `werror` [builtin option](Builtin-options.md#core-options).
+
+By default Meson uses as many concurrent processes as there are cores
+on the test machine. You can override this with the environment
+variable `MESON_TESTTHREADS`.
```

Comment:
Pursuant to my previous comment: documenting the use of MESON_TESTTHREADS in this context doesn't make this a clean naming scheme.